uqcsbot/scripts/weather.py

```diff
@@ -0,0 +1,214 @@
+from uqcsbot import bot, Command
+from urllib.request import urlopen
+import xml.etree.ElementTree as ET
+from datetime import datetime as DT
+
+
+def get_xml(state: str):
+    """
+    get BOM data as an XML for a given state
```

Comment:
Please capitalise the first word of the docstrings.

Reply:
Fixed.